src/Symfony/Component/Messenger/DependencyInjection/MessengerPass.php

```diff
@@ -184,6 +184,16 @@ private function registerHandlers(ContainerBuilder $container, array $busIds)
             }
             $container->getDefinition('console.command.messenger_debug')->replaceArgument(0, $debugCommandMapping);
         }
+
+        if ($container->hasDefinition('console.command.messenger_consume_messages')) {
+            $buses = array();
+            foreach ($busIds as $busId) {
+                $buses[$busId] = new Reference($busId);
+            }
+            $container->getDefinition('console.command.messenger_consume_messages')->replaceArgument(0, ServiceLocatorTagPass::register($container, $buses));
+            $container->getDefinition('console.command.messenger_consume_messages')->replaceArgument(4, $busIds);
```

Comment:
spare out an additional getDefinition call by chaining the 2nd `replaceArgument`?

Reply:
Must be the heatwave 😅